packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.test.tsx

```diff
@@ -193,8 +193,8 @@ describe('DetailsList', () => {
     expect(component).toBeDefined();
     (component as IDetailsList).focusIndex(2);
     setTimeout(() => {
-      expect(document.activeElement.querySelector('[data-automationid=DetailsRowCell]')!.textContent).toEqual('2');
-      expect(document.activeElement.className.split(' ')).toContain('ms-DetailsRow');
+      expect((document.activeElement as HTMLElement).querySelector('[data-automationid=DetailsRowCell]')!.textContent).toEqual('2');
```

Comment:
Personally I'd prefer `document.activeElement!` in cases like this. The `!` clearly communicates that the compiler thinks it's potentially null/undefined, whereas a cast could be done for a variety of reasons (and from a variety of starting points). Not a huge deal though and I'm open to other opinions.